vms/platformvm/add_validator_tx.go

```diff
@@ -34,10 +34,10 @@ import (
 
 var (
 	errNilTx                     = errors.New("tx is nil")
-	errWeightTooSmall            = errors.New("weight of this validator is too low")
-	errWeightTooLarge            = errors.New("weight of this validator is too large")
 	errStakeTooShort             = errors.New("staking period is too short")
 	errStakeTooLong              = errors.New("staking period is too long")
+	errWrongBondAmount           = errors.New("wrong bond amount for this validator")
+	errWeightTooSmall            = errors.New("weight of this validator is too low")
```

Comment:
Still needed?

Reply:
Yes, we validate sum of bond outs against fixed stake value, if its not equal - we send error

Reply:
I cannot find usage of errWeightTooSmall in the file.....

Reply:
My bad, thought you'r talking about `errWrongBondAmount`
`errWeightTooSmall` is used in `add_delegator_tx.go` (will be removed with delegation removal PR) and in `validator.go` `func (v *Validator) Verify()` - it returns err if weight is 0. Not sure if we should get rid of weight concept too (even if it will be the same for all validators for now), so I left this as it is.